httpserver/service_node.h

```diff
@@ -105,6 +105,8 @@ class ServiceNode {
     /// Process incoming blob of messages: add to DB if new
     void process_push_all(std::shared_ptr<std::string> blob);
 
+    bool is_pubkey_for_us(const std::string& pk);
```

Comment:
```suggestion
    bool is_pubkey_for_us(const std::string& pk) const;
```
I haven't followed the rabbit hole, so maybe you can't, but give it a try? You'll need to give the same treatment to `get_swarm_by_pk`